openwisp_firmware_upgrader/targets/ar71xx/generic/devices.py

```diff
@@ -0,0 +1,2 @@
+def returnData():
```

Comment:
The `camelCase` style is not a thing in python projects, not important though here, because I think we should get rid of the function and just leave something like `data = {....}`.

The data structure should be indented (should be doable using `openwisp-qa-format`, make sure to use the latest version available in the openwisp-utils repo).

Reply:
Thanks, I will look into it